src/server/ApiRoutes/Feedback.js

```diff
@@ -1,25 +1,65 @@
-import axios from 'axios';
+import aws from 'aws-sdk';
 
 import appConfig from '../../app/data/appConfig';
+import { encodeHTML } from '../../app/utils/utils';
 
 export default {
   post: (req, res) => {
     if (!req.body) return res.status(400).json({ error: 'Malformed request' });
     if (!req.body.fields) return res.status(400).json({ error: 'Request body missing `field` key' });
 
+    aws.config.update({ region: 'us-east-1' });
+
     const { fields } = req.body;
-    return axios({
-      method: 'POST',
-      url: appConfig.feedbackFormUrl,
-      data: {
-        fields,
+    const fullUrl = `${req.protocol}://${req.get('host')}${req.originalUrl}`;
```

Comment:
Isn't `req.originalUrl` always going to be "/research/collections/shared-collection-catalog/api/feedback"? I think you do need to POST the referring URL path to ensure you capture the relevant page. Just be sure to pass it through your `encodeHTML` function (to ensure no one is inserting malicious html).

Reply:
Oh interesting. I took that from the login redirect function. Now I am using `req.header.referer` passed through `encodeHTML`